packages/sst/src/constructs/SsrSite.ts

```diff
@@ -983,10 +1009,12 @@ function handler(event) {
     const cfDistributionProps = cdk?.distribution || {};
 
     const fnUrl = this.serverLambdaForRegional!.addFunctionUrl({
-      authType: FunctionUrlAuthType.NONE,
+      authType: !this.props.enableIAMAuth
```

Comment:
Super nit: The other patterns above does a truth check, while this one uses `!` as the first ternary.

Reply:
Fixed